Headers/QuartzCore/CALayer.h

```diff
@@ -207,6 +208,10 @@ NSString *const kCATransition;
 - (CGAffineTransform) affineTransform;
 - (void) setAffineTransform: (CGAffineTransform)affineTransform;
 
+@property (nonatomic, assign) CGColorRef borderColor;
+
+@property (nonatomic, assign) CGFloat contentsScale;
```

Comment:
Where do you synthesise these two variables?

Reply:
It is synthesized later. I'll just add a comment that `assign` is correct, too.

`contentsScale` is on the other hand correct, but not synthesized. I'm adding this. Presumably this worked because Clang.

I'd also prefer if this was moved elsewhere, up, with other animatable properties. I'll remedy this.